arabic/dream/another/dream2

```diff
@@ -0,0 +1,2 @@
+كنت أحاول الفرار من المجرمين وبينما أركض قلت لم لا أواجه هؤلاء الأوغاد فاستدرت فواجهتهم حتى قضيت على أربعة منهم وفر الباقى قبل أن ينادينى أحدهم لأسرع لأن ميعاد المدرسة سيفوتنى ان لم أستيقظ  
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.